azure/services/virtualmachines/virtualmachines.go

```diff
@@ -393,6 +403,10 @@ func (s *Service) generateStorageProfile(ctx context.Context, vmSpec azure.VMSpe
 		}
 	}
 
+	if vmSpec.UltraSSDEnabled {
+		sku.HasLocationCapabilities("UltraSSDAvailable")
```

Comment:
this is returning a bool, how is the output being used?

Reply:
added the validation now